Apps/CMakeLists.txt

```diff
@@ -49,9 +49,26 @@ else()
 endif()
 if(EXISTS ${PROJECT_SOURCE_DIR}/${ELASTICITY_SRC})
 
+  message(STATUS "Including ${PROJECT_SOURCE_DIR}/${ELASTICITY_SRC}")
```

Comment:
this is useful. can you do this for every module while at it?

Reply:
Done, by adding a macro to avoid too many repetitive code block.